MSAL/src/requests/MSALBaseRequest.m

```diff
@@ -29,19 +29,31 @@
 #import "MSALAuthority.h"
 #import "MSALHttpResponse.h"
 #import "MSALResult+Internal.h"
-#import "MSALTokenResponse.h"
 #import "MSALUser.h"
 #import "MSALWebAuthRequest.h"
 #import "MSALTelemetryAPIEvent.h"
 #import "MSIDTelemetry+Internal.h"
 #import "MSIDTelemetryEventStrings.h"
 #import "NSString+MSALHelperMethods.h"
 #import "MSALTelemetryApiId.h"
-#import "MSALClientInfo.h"
+#import "MSIDClientInfo.h"
 #import "NSURL+MSIDExtensions.h"
+#import "MSIDSharedTokenCache.h"
+#import "MSIDAccessToken.h"
+#import "MSALResult+Internal.h"
+#import "MSIDAADV2TokenResponse.h"
+#import "MSIDAADV2Oauth2Factory.h"
+#import "NSData+MSIDExtensions.h"
+#import "MSALErrorConverter.h"
 
 static MSALScopes *s_reservedScopes = nil;
 
+@interface MSALBaseRequest()
+
+@property (nullable, nonatomic) MSIDSharedTokenCache *tokenCache;
```

Comment:
just curious, where was cache coming from before?

Reply:
it was passes in MSALRequestParameters (same as it used to be in ADAL)